content/applications/inventory_and_mrp/inventory/product_management/configure/type.rst

```diff
@@ -2,230 +2,346 @@
+
+.. _inventory/product_management/BoM:
+
+On the product form for a tracked or untracked good, there are several smart buttons that may
+appear for manufacturing operations:
+
+- :icon:`fa-flask` :guilabel:`Bill of Materials`: The BoMs used to make this product.
+
+- :icon:`fa-level-up` :guilabel:`Used In`: Other goods that include this product in their BoM.
+
+.. _inventory/product_management/transfer-store:
+
+Transferring goods
+~~~~~~~~~~~~~~~~~~
+
+*Transfers* are warehouse operations that involve the movement of goods. Examples of transfers
+include
+:doc:`deliveries and receipts <../../shipping_receiving/daily_operations/receipts_delivery_one_step>`
+as well as :doc:`internal transfers <../../warehouses_storage/replenishment/resupply_warehouses>`
+between warehouses.
 
-   When the **Bill of Materials** and **Used In** smart buttons are visible on the product form,
-   this indicates the product can be manufactured or used as a component of a |BOM|.
+When creating a transfer for tracked products in the *Inventory* app, transfers modify the on-hand
+quantity at each location. For example, transferring five units from the internal location
+`WH/Stock` to `WH/Packing Zone` decreases the recorded quantity at `WH/Stock` and increases it at
+`WH/Packing Zone`.
+
+For untracked products, transfers can be created, but exact quantities at each storage location are
+not tracked.
 
 .. _inventory/product_management/package:
 
 Packages
---------
+~~~~~~~~
 
-Both storable and consumable products can be placed in :doc:`packages <package>`.
+Both tracked and untracked, or non-inventory, products can be placed in :doc:`packages <package>`.
 
-However, for consumable products, the quantity is not tracked, and the product is not listed in the
-package's :guilabel:`Contents`, accessed by going to :menuselection:`Inventory app --> Products -->
-Packages`, and selecting the desired package.
+However, for non-inventory products, the quantity is not tracked, and the product is not listed in
+the package's :guilabel:`Contents` (which can be accessed by going to :menuselection:`Inventory app
+--> Products --> Packages`, and selecting the desired package).
 
 .. figure:: type/package-content.png
    :align: center
    :alt: Show Packages page, containing the package contents list.
 
-   A consumable product was placed in the package, but the **Content** section does not list it.
+   An untracked product was placed in the package, but the **Content** section does not list it.
 
-If the *Move Entire Package* feature is enabled, moving a package updates the location of the
-contained storable products. However, the location of consumable products are not updated.
+Additionally, if the :guilabel:`Move Entire Packages` feature is enabled
+(:menuselection:`Inventory --> Configuration --> Operations Types`, and select any operation),
+moving a package updates the location of the contained tracked products but not the contained
```

Comment:
```suggestion
moving a package updates the location of the contained tracked products, but not the contained
```

Reply:
Comma not needed because it's not connecting two independent clauses